scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -75,17 +77,38 @@
           {% endif %}
             <td><input type="checkbox" value="{{variant._id}}" name="dismiss" onclick="enableDismiss()"></td>
             <td class="text-left">{{cell_rank(variant, institute, case, form, manual_rank_options)}}</td>
+            <td>{{ gene_cell(variant) }}</td>
+            <td>
+              {% for gene in variant.genes %}
+                {% for transc in gene.transcripts %}
+                  {% if transc.is_canonical %}
+                    <a href="http://grch{{ case.genome_build }}.ensembl.org/Homo_sapiens/Gene/Summary?t={{ transc.transcript_id }}" target="_blank">
+                        {{ transc.transcript_id }}
+                    </a>
+                  {% endif %}
+                {% endfor %}
+              {% endfor %}
+            </td>
+
+            <td>
+              {% for gene in variant.genes %}
+                {% if gene.exon %}
+                  <div>{{ gene.exon }}</div>
+                {% endif %}
+              {% endfor %}
+            </td>
             <td>
               <a href="{{ url_for('variant.variant', institute_id=institute._id, case_name=case.display_name,
                                   variant_id=variant._id, cancer='yes') }}">
-              {% for annotation in variant.canonical_transcripts %}
-                <div>{{ annotation }}</div>
-              {% endfor %}
+                {% for gene in variant.genes %}
```

Comment:
This sort of gets untenable for multiple genes. You will get a bunch of gene symbols in one column on the far left, and a bunch of hgvs strings on the right. Pairing is anyones guess. I would suggest either adding in a "symbol:" like for the consequences, or to save space keep a strict line order within the cells so one can read the row of gene->transcript->exon->hgvs->consequence. Perhaps group them in columns next to one another as well to make it easy to read?

Reply:
![Screenshot 2021-02-25 at 12 07 54](https://user-images.githubusercontent.com/758570/109144934-1eabad80-7762-11eb-8785-c4d2e78804b4.png)
And then stretch the window size to watch the symbols and transcripts reorder.. 😊